packages/cli/src/commands/new/project.ts

```diff
@@ -69,13 +73,20 @@ export default class Project extends Command {
   }
 }
 
-const run = async (flags: Partial<ProjectInitializerConfig>, boosterVersion: string): Promise<void> =>
-  Script.init(`boost ${Brand.energize('new')} 🚧`, parseConfig(new Prompter(), flags, boosterVersion))
+const run = async (flags: Partial<ProjectInitializerConfig>, boosterVersion: string): Promise<void> => {
+  const boostScript = Script.init(`boost ${Brand.energize('new')} 🚧`, parseConfig(new Prompter(), flags, boosterVersion))
     .step('Creating project root', generateRootDirectory)
     .step('Generating config files', generateConfigFiles)
+
+  if (flags['skip-install']) {
+    return boostScript.info('Project generated!').done()
+  }
+
+  return boostScript
     .step('Installing dependencies', installDependencies)
     .info('Project generated!')
     .done()
+}
```

Comment:
Lovely! Perhaps we can make this a bit shorter like:
```suggestion
  Script.init(`boost ${Brand.energize('new')} 🚧`, parseConfig(new Prompter(), flags, boosterVersion))
    .step('Creating project root', generateRootDirectory)
    .step('Generating config files', generateConfigFiles)
    .step(
      'Installing dependencies', 
      (config: BoosterConfig) => flags.skipInstall ? undefined : installDependencies(config)
    )
    .info('Project generated!')
    .done()
}
```

Reply:
Tried to do this, but `step` method will print message that it is installing dependencies. 
Looks like we need to change `step` method to skip this step if action returns undefined